src/harness/reference_models/dpa/move_list.py

```diff
@@ -252,6 +252,8 @@ def computeInterference(grant, constraint, inc_ant_height, num_iteration, dpa_ty
 
   # Calculate the interference contributions
   interf = eirp_cbsd - path_loss
+  if np.isscalar(interf):
```

Comment:
I dont think it works, because above we have:
  `path_loss = np.array(results.db_loss)`

And if results.db_loss is scalar 0, then path_loss is an 'array scalar', not a scalar.
Try this in ipython for example:
```
In [25]: a= 0

In [26]: np.array(a)
Out[26]: array(0)

In [27]: np.isscalar(np.array(0))
Out[27]: False
```
So there are 2 easy solution to the issue:
1) change above in line 228:
```
   if np.isscalar(results.db_loss):
     path_loss = np.full(num_iteration + 1, results.db_loss)
   else:
     path_loss = np.array(results.db_loss)
``` 
2) fix it in wf_itm.py directly, which is more clean and robust on other client code using the lib:
```
  # Case of same points
  if (lat_cbsd == lat_rx and lon_cbsd == lon_rx):
    return _PropagResult(
        db_loss = 0 if np.isscalar(reliability) else [0] * len(reliability),
        incidence_angles = _IncidenceAngles(0,0,0,0),
        internals = None)  
```

Reply:
Serga, thank for your usefull comment!, I updated it, please check.